fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergApiConverter.java

```diff
@@ -398,4 +408,83 @@ public static void applySchemaChanges(UpdateSchema updateSchema, List<AlterClaus
 
         updateSchema.commit();
     }
+
+    // modify table comment/properties/name
+    public static void applyTableChanges(Transaction transaction,
+                                         IcebergCatalog icebergCatalog,
+                                         String dbName,
+                                         String tableName,
+                                         List<AlterClause> tableChanges) {
+        if (transaction == null || transaction.table() == null) {
+            throw new StarRocksConnectorException("Transaction or table cannot be null");
+        }
+
+        for (AlterClause clause : tableChanges) {
```

Comment:
Could you use a design pattern to avoid a lot of if else?

Reply:
sure